index.js

```diff
@@ -1,85 +1,108 @@
+/* globals define, document, module */
 'use strict';
 
-var util = require('./util.js');
-
-var body = document.querySelector('body');
+(function (name, context, definition) {
```

Comment:
Is this something we need at eBay? I'm trying to keep these modules minimal for now, and so only serving the eBay use case. However if this is something that is strongly recommended, we can go and update all other makeup modules to this pattern.

Reply:
I wouldn't say it's _strongly_ recommended, but I thought it would be a nice addition. Then it could be used by any site, not simply a node one. So if the focus is for eBay only, then it will work just fine as is.